astroid/__init__.py

```diff
@@ -69,6 +62,8 @@
 
 # more stuff available
 from astroid import raw_building
+
+from astroid.const import Context, Load, Store, Del
```

Comment:
Do we want to add `Context` here? `Load`, `Store`, and `Del` are required so we don't break stuff but I feel like it might make sense to not add any more. We could even think about removing all here for the next major release. It shouldn't be a big issue to import it directly from `astroid.const`. What do you think?

The spirit of `astroid` so far has been to export everything from `__init__`, but that's obviously causing issues now.

Reply:
I think the import from astroid init directly are not a problem as long as    it's in pylint. It actually permits to change the structure in astroid without creating breaking changes. In pylint an import directly from a module must even be avoided as a change in structure would break it.

 But in astroid itself we can't do that as it's creating cyclic import and more problem than what an elegant import solves. (Runtime definition of variable or function in some class). And if we're refactoring a module in astroid we're aware of it so changing the import is not costly at all (my ide handles everything).

Reply:
So yes I think we should create an API with everything in ``astroid.__init__.py`` so we can refactor astroid without affecting the tools using it.

Reply:
> So yes I think we should create an API with everything in `astroid.__init__.py` so we can refactor astroid without affecting the tools using it.

Ok. That does make sense.